src/notation/view/widgets/editstyle.ui

```diff
@@ -222,6 +222,11 @@
         <string>Chord Symbols</string>
        </property>
       </item>
+      <item>
+       <property name="text">
+        <string>Chord Symbols New</string>
```

Comment:
Strange naming... I guess this is only temporary?

Reply:
Yes. In the end, I will take out the old Chord Symbols page and rename this.

Reply:
Done